judges/quantity-of-deliverables/quantity-of-deliverables.rb

```diff
@@ -53,4 +53,13 @@
   end
   f.total_issues_created = issues
   f.total_pulls_submitted = pulls
+
+  # Total number of releases published in all repositories from since
+  releases = 0
+  Fbe.unmask_repos.each do |repo|
+    Fbe.octo.releases(repo).each do |json|
+      releases += 1 if !json[:draft] && !json[:published_at].nil? && json[:published_at].utc > f.since.utc
+    end
+  end
```

Comment:
@Yegorov can we directly count the number of releases that meet the criteria using the `count` method?
like this:
```suggestion
  releases = Fbe.unmask_repos.sum do |repo|
    Fbe.octo.releases(repo).count do |json|
      !json[:draft] && json[:published_at] && json[:published_at].utc > f.since.utc
    end
  end
```
What do you think?

Reply:
@Suban05 Yes of course, It will look better like this:
```ruby
  f.total_releases_published =
    Fbe.unmask_repos.sum do |repo|
      Fbe.octo.releases(repo).count do |json|
        !json[:draft] && json[:published_at] && json[:published_at].utc > f.since.utc
      end
    end
```
new line after assignment because of Rubocop: `Layout/MultilineAssignmentLayout: Right hand side of mult
i-line assignment is on the same line as the assignment operator =`

@yegor256 What do you think?

Reply:
@Yegorov this is how Rubocop wants it and it's correct

Reply:
@Yegorov I also believe that `json[:published_at] > f.since` will provide the same result. No need to convert to UTC. I'm not 100% sure though...

Reply:
@yegor256 Yes, most likely in Ruby this work
```ruby
3.3.4 :011 > Time.parse('2024-08-15 09:00:00 MSK') > Time.parse('2024-08-15 06:00:00 UTC')
 => false 
3.3.4 :012 > Time.parse('2024-08-15 09:00:01 MSK') > Time.parse('2024-08-15 06:00:00 UTC')
 => true 
```

Fixed